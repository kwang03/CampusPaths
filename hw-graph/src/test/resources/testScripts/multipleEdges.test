#Tests the behavior of graph when there are multiple edges between two nodes

#Creates graph
CreateGraph myGraph

#Adds some nodes to the graph
AddNode myGraph n1
AddNode myGraph n2

#Adds edges from n1 to n2 and lists children
AddEdge myGraph n1 n2 e1
ListChildren n1

AddEdge myGraph n1 n2 e2
ListChildren n1

AddEdge myGraph n1 n2 e3
ListChildren n1

AddEdge myGraph n1 n2 e4
ListChildren n1

#Checks that n2 has no children
ListChildren n2