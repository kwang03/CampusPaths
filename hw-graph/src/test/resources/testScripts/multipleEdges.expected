#Tests the behavior of graph when there are multiple edges between two nodes

#Creates graph
created graph myGraph

#Adds some nodes to the graph
added node n1 to myGraph
added node n2 to myGraph

#Adds edges from n1 to n2 and lists children
added edge e1 from n1 to n2 in myGraph
the children of n1 in myGraph are: n2(e1)

added edge e2 from n1 to n2 in myGraph
the children of n1 in myGraph are: n2(e1) n2(e2)

added edge e3 from n1 to n2 in myGraph
the children of n1 in myGraph are: n2(e1) n2(e2) n2(e3)

added edge e4 from n1 to n2 in myGraph
the children of n1 in myGraph are: n2(e1) n2(e2) n2(e3) n2(e4)

#Checks that n2 has no children
the children of n2 in myGraph are: